packages/relay/tests/lib/subscriptionController.spec.ts

```diff
@@ -252,5 +260,7 @@ describe('subscriptionController', async function () {
     const subId2 = subscriptionController.subscribe(wsConnection, tag1.event);
 
     expect(subId).to.be.eq(subId2);
+    subscriptionController.unsubscribe(wsConnection, subId);
+    subscriptionController.unsubscribe(wsConnection, subId2);
```

Comment:
Can we move all `subscribe` and `unsubscribe` calls in `beforeEach` and `afterEach`?

Reply:
Well, they are not all the same, so Im not sure whats the point of moving them all?
What can be moved is the creation of the MockWsConnection instantiation, but in the different tests we subscribe to different tags/events etc. :?

Reply:
Sorry, you are right. I thought we could extract them somehow in order to ensure that we always call unsubscribe on all subscribed events/tags at the end of the test. Let's keep it as is.